learn-to-use-vim/06-combining-vim-commands.md

```diff
@@ -0,0 +1,18 @@
+# Combining Vim commands
+
+**[📹 Video](https://egghead.io/lessons/misc-combining-vim-commands)**
+
+## Building commands
+
+Vim commands are built in three parts:
+
+- an action
+- a location
+- a second location (context)
+
+Vim sees tests as an object that can be manipulated.
+
+## Key Combo Trick
+
+- Delete text in between <div> tags by using the keys `C`-`I`-`T`.
```

Comment:
Here I think you need backticks around the `<div>` or else it creates a new line.